.github/workflows/docs-preview.yml

```diff
@@ -0,0 +1,15 @@
+name: Preview docs on Netlify
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - docs/**
+
+jobs:
+  preview:
+    uses: apollographql/docs/.github/workflows/preview.yml@main
```

Comment:
I think using `main` is fine for now, but just as a thought, you might find that it's a bit more durable (and allows you to make temporary mistakes on `main` without ripple effects throughout every docs providing repository) to employ semantic versioning here.  I think these are the workflows that GitHub Actions themselves suggest?

- https://github.com/github-developer/javascript-action/blob/963a3b9a9c662fd499419a240ed8c49411ff5add/.github/workflows/publish.yml; which uses:
- https://github.com/JasonEtco/build-and-tag-action